src/abstractdataframe/abstractdataframe.jl

```diff
@@ -390,20 +390,23 @@ describe(io, df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :nmissi
 * `io` : optional output descriptor
 * `stats::AbstractVector{Symbol}`: the summary statistics to report. Allowed 
   fields are `:mean`, `:std`, `:min`, `:q25`, `:median`, `:q75`, `:max`, `:eltype`, 
-  `:nunique`, and `:nmissing`
+  `:nunique`, `:first`, `:last`, and `:nmissing`
 
 **Result**
 
 * A `DataFrame` where each row represents a variable and each column a summary statistic.
 
 **Details**
 
-If the column's base type derives from `Number`, compute the mean, standard 
-deviation, minimum, first quantile, median, third quantile, and maximum. If 
-a column is not numeric, these statistics are populated with `nothing`s. 
+* If the column's base type derives from `Real`, compute the mean, standard 
```

Comment:
"a column". Could also just say "For `Real` columns" on so on below. But maybe you could just say that all these stats are computed if possible, as you say for the fallback?

Actually, do we really need special methods for `Real` now? They should give the same result, there are just additional `try... catch` checks which don't really hurt, right?

Reply:
The only difference would be `unique()`, which might be slow. I will do some benchmarking to see if it really matters

Reply:
But that's not julian! Only multiple dispatch!

I will make that addition. It definitely makes things simpler.

Reply:
fixed.